meinberlin/apps/votes/templates/meinberlin_votes/voting_code_dashboard.html

```diff
@@ -0,0 +1,40 @@
+{% extends "a4dashboard/base_dashboard_project.html" %}
+{% load i18n %}
+
+{% block dashboard_project_content %}
+<h1 class="u-first-heading mt-0">{% translate 'Generate voting codes' %}</h1>
+
+<p class="form-hint">{% blocktranslate %}For voting in the third phase, participants need a voting code. You can generate and download the codes here. {% endblocktranslate %}</p>
+<p>{% blocktranslate count counter=number_of_module_tokens %}
+One code has been generated for this module so far.
```

Comment:
out of curiosity. with adding the voting module it will generate one code per se?

Reply:
No, you are right, we should use the number here as well as in other languages the 0 might have an extra form. In English and German the plural is used, so it should be fine. I will check to be sure, though!

Reply:
Yes, it does work with the 0 in English and it will also be fine in German with the translation